src/services/assetService.ts

```diff
@@ -107,6 +108,7 @@ export class AssetService {
         return assets.map((asset) => {
             if (!asset.path.toLowerCase().startsWith("http://") && !asset.path.toLowerCase().startsWith("https://")) {
                 asset.path = "file:" + asset.path;
```

Comment:
Unrelated: Does the `file:` need the `//` chars so it is like `file://` ??  I think its all working right now so I guess it does not.

Reply:
Yeah, file: by itself just looks weird to me, but otherwise lgtm.